### PR TITLE
#CP-11 sign in page fixes

### PR DESCRIPTION
**Pull request by Yvad60**
#### What the PR does
Fix bugs of the landing page

#### Description of Tasks to be completed

- [x] Responsiveness is improved
- [x] Users should see different states for interactive elements
- [x] Skeleton UIs/screen are/is shown when the page loads/is loading
- [x] All tests are written

#### How this should be manually tested
Clone the project repository linked [here](https://github.com/atlp-rwanda/phantom-fe-codebandits)

```
cd phantom-fe-codebandits/
git checkout bg-sign-in-page-fixes-cp-11
git pull origin bg-sign-in-page-fixes-cp-11
npm install
npm run start
run http://localhost:3000 in your browser
```

### CODE REVIEW

**Review by samyshyaka — CHANGES_REQUESTED** [review]
This is great work Yvad, 
On the medium and smaller screens, try to make the top part of the image behind the card a little more visible

**Review by feyton — DISMISSED** [review]
Great work @Yvad60 the responsiveness and test coverage has been improved.

**Review by samyshyaka — CHANGES_REQUESTED** [review]
On larger screens, try to center the image on y-axis. 
on smaller screens make the image, a little bit more visible at the top.

File: src/pages/Login.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ import { useNavigate } from 'react-router-dom';
 import { toast } from 'react-toastify';
 import busMapImg from '../assets/busMap.png';
```
Comment (Abdulkeza) [inline src/pages/Login.js]: Try to add space between built-in modules and your own module

File: src/pages/__test__/Login.test.js [hunk 2]
```diff
@@ -15,3 +29,75 @@ describe('Login', () => {
...
+  });
+  it('has one h4 element', () => {
+    expect(wrapper.find('h4').length).toEqual(1);
+  });
+  it('has forgot password heading', () => {
+    expect(wrapper.find('h4').text()).toEqual('Forgot password?');
+  });
+});
+
+describe('Landing page functionality tests', () => {
```
Comment (Abdulkeza) [inline src/pages/__test__/Login.test.js]: Are you testing a landing page??

**Comment by Yvad60** [comment]
> On larger screens, try to center the image on y-axis. on smaller screens make the image, a little bit more visible at the top.

yeah!

**Review by samyshyaka — APPROVED** [review]

**Pull request merged**